PWGHF/TableProducer/candidateCreator2Prong.cxx

```diff
@@ -185,19 +190,24 @@ struct HfCandidateCreator2Prong {
     setLabelHistoCands(hCandidates);
   }
 
-  template <bool doPvRefit, o2::hf_centrality::CentralityEstimator centEstimator, typename Coll, typename CandType, typename TTracks>
+  template <bool doPvRefit, bool useUpcSel = false, o2::hf_centrality::CentralityEstimator centEstimator, typename Coll, typename CandType, typename TTracks, typename BCsType>
```

Comment:
```suggestion
  template <bool doPvRefit, bool useUpcSel, o2::hf_centrality::CentralityEstimator centEstimator, typename Coll, typename CandType, typename TTracks, typename BCsType>
```
Don't use default template arguments, please. The behaviour should be obvious from the call.